source/boards/data-and-archives.rst

```diff
@@ -0,0 +1,106 @@
+Import your data
+================
+
+You can import data from other tools to use with Boards.
+
+Import from Asana
+-----------------
+
+This node app converts an Asana JSON archive into a ``.focalboard`` archive.
+
+1. Log into your Asana account.
+2. From the Asana Board Menu, select **Export/Print > JSON** to create an archive file (which you'll use in Boards).
+3. Save the file locally, e.g. to ``asana.json``.
+4. In your local copy of Boards, open a terminal window.
+5. Navigate to ``focalboard/webapp``.
+6. Run ``npm install``.
+7. Change directory to ``focalboard/import/asana``.
+8. Run ``npm install``.
+9. Run ``npx ts-node importAsana.ts -i <asana.json> -o archive.focalboard``.
```

Comment:
Is this still in ``focalboard/import/asana``? Is there any expected output?

Reply:
Yes that is the correct path.  We display the Board and Card titles that get imported. As well as a summary.
```
Scotts-MacBook-Pro:asana sbishel$ npx ts-node importAsana.ts -i ~/Downloads/asana.json -o archive.focalboard
Board: 1:1 Meeting Agenda Test
Card: [READ ME] Instructions for using this project
Card: [EXAMPLE TASK] Feedback on design team presentation
Card: [EXAMPLE TASK] Finalize monthly staffing plan
Card: [EXAMPLE TASK] Review Q2 launch video outline
Card: [EXAMPLE TASK] Mentor a peer

Found 5 card(s).
Exported to archive.focalboard

```